soh/src/overlays/actors/ovl_En_Ds/z_en_ds.c

```diff
@@ -106,6 +106,7 @@ void EnDs_TalkAfterBrewOddPotion(EnDs* this, PlayState* play) {
         u32 itemId = GI_ODD_POTION;
         if (GameInteractor_Should(VB_TRADE_ODD_MUSHROOM, true, this)) {
             Actor_OfferGetItem(&this->actor, play, itemId, 10000.0f, 50.0f);
+            Flags_SetItemGetInf(ITEMGETINF_30);
```

Comment:
Any reason this was added?
If it's from decomp then no problem but if this is ours, it'd be better to set it on the call to `VB_TRADE_ODD_MUSHROOM`.

Reply:
I think this may have been a vestigial thing, will remove it and retest granny things